manifests/profile/apt_mirror.pp

```diff
@@ -0,0 +1,54 @@
+# Copyright (c) 2021 The Regents of the University of Michigan.
+# All Rights Reserved. Licensed according to the terms of the Revised
+# BSD License. See LICENSE.txt for details.
+
+# nebula::profile::apt_mirror
+#
+# Profile for apt repo mirror
+#
+# @example
+#   include nebula::profile::apt_mirror
+class nebula::profile::apt_mirror (
+) {
+
+  package {
+    [ 'debmirror', 'debian-keyring', 's3fs' ]:
+  }
+
+  group { 'apt-mirror':
+    ensure => 'present',
+    system => 'true',
+    gid    => '996',
+  }
+
+  user { 'apt mirror user':
+    ensure     => 'present',
+    name       => 'apt-mirror',
+    comment    => 'apt mirror user',
+    gid        => '996',
+    groups     => ['apt-mirror'],
+    home       => '/home/apt-mirror',
+    password   => '!',
+    shell      => '/bin/bash',
```

Comment:
Shouldn't this user's shell be `/bin/false`?

Reply:
Let me check that. It may be a leftover bit from when I was testing and isn't actually needed anymore.

Reply:
Updated this.